README.md

```diff
@@ -66,9 +66,50 @@ contract MyContract is Ownable {
+- **token** - A collection of approved ERC standard tokens -- their interfaces and implementations.
+- **ERC20** - A standard interface for fungible tokens:
+	- *Interfaces* - Includes the ERC-20 token standard basic interface. I.e., what the contract’s ABI can represent.
+	- *Implementations* - Includes ERC-20 token implementations that include all required and some optional ERC-20 functionality.
+- **ERC721** - A standard interface for non-fungible tokens 
+	- *Interfaces* - Includes the ERC-721 token standard basic interface. I.e., what the contract’s ABI can represent.
+	- *Implementations* - Includes ERC-721 token implementations that include all required and some optional ERC-721 functionality.
+
+## Tests
+Unit test are critical to the OpenZeppelin framework. They help ensure code quality and mitigate against security vulnerabilities. The directory structure within the `/tests` directory corresponds to the `/contracts` directory. OpenZeppelin uses Mocha’s JavaScript testing framework and Chai’s assertion library. To learn more about how to tests are structured, please reference OpenZeppelin’s Testing Guide.
+
+## How To Use And Modify OpenZeppelin Contracts
+When using OpenZeppelin to build your own distributed applications, for security reasons we encourage you NOT to modify the framework’s base contracts, libraries, and interfaces. In order to leverage and extend their functionality, we encourage you to inherit from them.
+
+The Solidity programming language supports multiple inheritance. This is very powerful yet it can also be confusing. The more complexity you introduce to your distributed applications through multiple inheritance, the greater your application’s attack surface is.
+
+You’ll notice in the `/mocks` directory there are a collection of abstract contracts that can be used as the foundation for your own custom implementations. These mock contracts demonstrate how OpenZeppelin’s secure base contracts can be used with multiple inheritance.
+
+To learn more about combining OpenZeppelin contracts with your own custom contracts using multiple inheritance we encourage you to read the following: [On crowdsales and multiple inheritance](https://blog.zeppelin.solutions/on-crowdsales-and-multiple-inheritance-af90c694e35f)
+
 ## Security
 OpenZeppelin is meant to provide secure, tested and community-audited code, but please use common sense when doing anything that deals with real money! We take no responsibility for your implementation decisions and any security problem you might experience.
 
+The core development principles and strategies that OpenZeppelin is based on include: security in depth, simple and modular code, clarity-driven naming conventions, comprehensive unit testing, pre-and-post-condition sanity checks, code consistency, and regular audits.
```

Comment:
Maybe add a link to the wiki? https://github.com/OpenZeppelin/openzeppelin-solidity/wiki/Contribution-guidelines

The wiki will be deprecated eventually though, so I'm not sure how to proceed here.